stored_requests/backends/db_provider/mysql_dbprovider_test.go

```diff
@@ -83,7 +83,7 @@ func TestConnStringMySql(t *testing.T) {
 			cfg: cfg,
 		}
 
-		connString := provider.ConnString()
+		connString, _ := provider.ConnString()
```

Comment:
Can we add a few additional test cases to exercise the new logic in `ConnString` and its helper function `setupTLSConfig`? It looks like we would just need to generate a couple of fake certificates.